main.tex

```diff
@@ -245,20 +244,11 @@ \section{Methods}
 and restricted memory.
 
 \section{Results}\label{section:results}
-% \subsection{Best response memory-one strategies}\label{section:best_response_memory_one}
 
-As discussed ZDs have been acclaimed for their robustness
-against a single opponent. ZDs are evidence that extortion works
+ZDs are evidence that extortion works
 in pairwise interactions, their behaviour ensures that the strategies will never
```

Comment:
This sentence feels a bit standalone. 

> ZDs are evidence that extortion works
in pairwise interactions, their behaviour ensures that the strategies will never
lose a game.

> Now we are going to define best response memory-one strategies

Reply:
I think I agree, I've removed it.